pybamm/models/full_battery_models/lithium_ion/base_lithium_ion_model.py

```diff
@@ -85,6 +85,29 @@ def set_sei_submodel(self):
         # positive electrode
         self.submodels["positive sei"] = pybamm.sei.NoSEI(self.param, "Positive")
 
+    def set_li_plating_submodel(self):
```

Comment:
in general I don't think anything is lost by calling all these things `lithium_plating`, in fact makes it easier to read